frontend/index.js

```diff
@@ -109,6 +109,7 @@ function makeChange(doc, requestType, context, options) {
 
   if (doc[OPTIONS].backend) {
     const [backendState, patch] = doc[OPTIONS].backend.applyLocalChange(state.backendState, request)
+    if (options && options.patchCallback) options.patchCallback(patch)
```

Comment:
I would prefer a operate line + indentation, which improves readability, safety and debuggablility.